client/mysql.cc

```diff
@@ -3786,9 +3786,10 @@ print_table_data_html(MYSQL_RES *result)
   MYSQL_FIELD	*field;
 
   mysql_field_seek(result,0);
-  (void) tee_fputs("<TABLE BORDER=1><TR>", PAGER);
+  (void) tee_fputs("<TABLE BORDER=1>", PAGER);
   if (column_names)
   {
+	(void) tee_fputs("<TR>", PAGER);
```

Comment:
Hi chris, can you please indent this line ?

Reply:
Hi Anel, I've indented it properly now (I mistakenly used a tab instead of spaces).  However, I am having trouble merging this into 1 single commit.  I can start from scratch and redo this if you like, so that it is a single commit.  Please advise when you have a chance.  Thank you.

To give some added insight into what I did, I first tried to modify it in my local branch on my computer.

The first command seems to be correct:

C:\wpafb\hangar18\10.3\server>git checkout 10.3-MDEV-15538 & git rebase 10.3
Already on '10.3-MDEV-15538'
Current branch 10.3-MDEV-15538 is up to date.

However, the second is where I see a problem:

C:\wpafb\hangar18\10.3\server>git rebase -i HEAD~2
Cannot rebase: You have unstaged changes.
Please commit or stash them.

Then the 3rd command (git push --force https://github.com/chriscalender/server/ 10.3-MDEV-15538) gave the same error.

I even tried to commit again, but the change did not appear when I checked the file online.

Thus, I modified the file online.  This does lead to 2 commits.  However, I don;t know how to merge them into 1.  Perhaps I cannot merge a local fix with an online fix?  or perhaps I'm just overlooking something?  I'm kind of stuck on trying to get this into 1 commit, unless I start from scratch.

Reply:
Hi Chris,
thank you for updating this one :).
No need to open new PR for this, you already pushed additional commit.
In order to rebase this try to do the following:
1) Checkout to your developing branch 
(I suppose you have as a last 2 commits `6ec39dda2b5d9e3276c75aaf2a6f8dd838505bdb` and `07755b29d7bc12e5d97f78030172ed6c8e211d1f`) and reset all changes that you did before.
`git checkout 10.3-MDEV-15538 && git reset --hard HEAD`
2) Now try to rebase last 2 commits like you did: `git rebase -i HEAD~2` use squash for the last commit and pick for the last-1 and leave the same commit message
3) Now when everything is rebased use force pushed like you already did recently :).
Please note if you have additional questions try to use our zulip channel intended for chat `https://mariadb.zulipchat.com/#`